README.md

```diff
@@ -6,7 +6,22 @@
 
 ---
 
-The IntelliJ Ballerina plugin provides the Ballerina development capabilities in IntelliJ IDEA. Click the links below for instructions on how to download, install, and use the features of the IntelliJ plugin.
+## Overview
+The Ballerina IntelliJ plugin provides the [Ballerina programming language](https://ballerina.io/) development capabilities within IntelliJ IDEA. While it offers a subset of features compared to the [Ballerina VSCode extension](https://marketplace.visualstudio.com/items?itemName=WSO2.ballerina), it is designed to meet the specific needs of those working within the IntelliJ ecosystem.
+
+> **Note:** This plugin is being developed as an experimental and community-driven project. The plugin is fully open-source, and we welcome contributions!
+
+## Intended Users
+
+This plugin is mainly targeted at **Ballerina library developers** who seek:
+- Robust interoperability between Ballerina and Java
+- A native development experience within IntelliJ IDEs
+
+Therefore, if you do not fall into the above categories, and looking for a more feature-rich experience (including graphical programming capabilities) with Ballerina, we recommend using the official [Ballerina VSCode extension](https://ballerina.io/learn/vs-code-extension/).
+
+## Plugin Guide
+
+Click the links below for instructions on how to download, install, and use the features of the IntelliJ plugin.
```

Comment:
Are these pages up-to-date?

Reply:
Yes we have updated these pages as a part of the same effort. Please let me know incase you found anything outdated.